src/turbolinks/cache.coffee

```diff
@@ -9,12 +9,12 @@ class Turbolinks.Cache
 
   get: (location) ->
     return unless @has(location)
-    snapshot = @read(location)
+    snapshot = @read(location).clone()
```

Comment:
After thinking about this some more, I'm not entirely sure *this* clone is necessary.